XenModel/Messages.resx

```diff
@@ -4245,6 +4245,9 @@ For optimal performance and reliability during VM migration, ensure that the net
   <data name="DATEFORMAT_DM" xml:space="preserve">
     <value>MMM d</value>
   </data>
+  <data name="DATEFORMAT_DM_HM" xml:space="preserve">
+    <value>MMM d, h:mm:ss</value>

```

Comment:
This format should need AM/PM indicator, better switch to 24h format. Also, do we need the year? (Theoretically someone can start the update, go out to the New Year's countdown party, and come back afterwards to resume work).

Reply:
> This format should need AM/PM indicator, better switch to 24h format.
👍 
> Also, don't we need the year? (Theoretically someone can start the update, go out to the New Year's countdown party, and come back afterwards to resume work).

Thought about it but I thought the extra verbosity of the format wasn't worth covering this edge case. I don't think it'd be confusing, especially if you're running it around New Year's 😅